docs/usage/collect-metrics.md

```diff
@@ -0,0 +1,31 @@
+# Collect metrics
+
+This section presents the metrics exposed by **liqo components**, which can be scraped by [Prometheus](https://prometheus.io/). In order to show how to use those metrics some [Grafana](https://grafana.com/) dashboards are provided.
+
+## Prerequisites
+
+In order to collect metrics from Liqo components, you need to install Prometheus and Grafana.
+
+It's suggested to install the **kube-prometheus** stack to deploy them. Instructions can be found [here](https://github.com/prometheus-operator/kube-prometheus)
+
+## How to enable metrics
+
+Metrics are **disabled** by default. In order to allow Prometheus to scrape metrics from liqo components, you have to enable metrics by setting the `metrics.enabled` value to `true` in the `values.yaml` file. To enable the creation of **ServiceMonitor** resources for Prometheus, you can set the `metrics.serviceMonitor.enabled` value to `true`.
```

Comment:
* Please, stick to one sentence per line, in order to limit the issues with git versioning.
* The standard way to enable metrics collection is to use the appropriate liqoctl flag, but you did not mention that here. You could also mention that advanced aspects can be customized through the helm values file.